index.js

```diff
@@ -28,20 +31,35 @@ function patchFlag(patch, key, config, cb) {
 }
 
 const fetchFlags = function (config, cb) {
-  const { baseUrl, projectKey, sourceEnvironment, destinationEnvironment, apiToken } = config;
+  const { baseUrl, projectKey, sourceEnvironment, destinationEnvironment, apiToken, tags, flag } = config;
+  let isSingle = flag ? true : false;
```

Comment:
This can be simplified to:

```ts
let isSingle = !!flag
```

Reply:
i modified this a but to make sure tags aren't included (which they shouldn't be)